docker-compose.yaml

```diff
@@ -22,49 +21,26 @@ services:
       IN_DOCKER: 'true'
     restart: on-failure:5
     mem_swappiness: 0
-    security_opt:
-      - no-new-privileges
-    volumes:
-      - ./config:/app/config/:ro
-      - ${MONITOR_DATA_DIR:-/dev/null}:/app/data
-      - ${LOG_DATA_DIR:-./logs}:/app/logs
-    networks:
-      - monitor-network
-      - metrics-network
-
+    security_opt: [no-new-privileges]
+    volumes: [./config:/app/config/:ro, '${MONITOR_DATA_DIR:-/dev/null}:/app/data', '${LOG_DATA_DIR:-./logs}:/app/logs']
+    networks: [monitor-network, metrics-network]
```

Comment:
Would be great to avoid flow style fixes here. May be to revert and directly update yamlfix to `keep_style`